spec/unit/spotlight/dor/indexer_spec.rb

```diff
@@ -25,27 +28,211 @@
       'SC0340, Accession 2005-101',
       'SC0340, Accession 2005-101, Box 18'
     ]
-    parsable_exemplars_based_on_actual_data.each do |example|
-      it "parses series number from '#{example}'" do
-        ng_mods = Nokogiri::XML("#{mods_start}#{example}#{mods_end}")
+    exemplars_based_on_feigenbaum.each do |example|
```

Comment:
ok.  I think I can adopt your suggestion in the next PR (already started coding) and you can trust me to revise the tests as part of that PR.  I'm not a fan of accruing tech debt when the knowledge is already loaded into memory.  

But we can decide tomorrow.